src/enums/roleTypes.ts

```diff
@@ -1,10 +1,9 @@
 export enum RoleTypes {
-  DIRECTOR = 'Director',
   COMPLETING_PARTY = 'Completing Party',
   CUSTODIAN = 'Custodian',
+  DIRECTOR = 'Director',
   INCORPORATOR = 'Incorporator',
   LIQUIDATOR = 'Liquidator',
   PARTNER = 'Partner',
   PROPRIETOR = 'Proprietor',
-  SUBSCRIBER = 'Subscriber',
```

Comment:
Not used in this UI and did not match [party schema](https://github.com/bcgov/business-schemas/blob/7be2b1722e015202d65c55e838daff2f132b3625/src/registry_schemas/schemas/party.json#L74), so I deleted this.